### ISSUE TITLE
react-native-fbsdk support

### ISSUE THREAD

**Issue by omrihaim** [expo/expo#2376]
Expo init and snack.expo.com don't support react-native-fbsdk so installing react-native-fbsdk ends in invariant Violation: View config not found for name RTCFBLoginButton.

Just try to use the Login Button flow as facebook team suggest in the following article:
https://developers.facebook.com/docs/react-native/login/
in an empty snack project and you will immediately get this issue. 

Similar problem occur while doing the same in a project started using expo init.

**Comment by tcdavis** [comment]
That's expected behaviour since react-native-fbsdk includes native code (Any library that includes a `react-native link` step in its installation instructions)

Fortunately, Expo has a Facebook module that should cover your needs.  Here's a snack demonstrating how to use it: https://snack.expo.io/Sk2jBj-Wf

**Comment by omrihaim** [comment]
Thank you, but this isn't the solution I was looking for, as unlike the Facebook Button that gives you the functionality to open the facebook app, this is a wrapper for the web/js api so give different set of capabilities. 
I think this issue need to be re-open, as it is important to solve it for better facebook integration.

**Comment by EvanBacon** [comment]
TL;DR: I added [native login](https://github.com/expo/expo/blob/03220c86e2c07a88aeec6b40f2fac193cbcebfa4/ios/Exponent/Versioned/Core/Api/EXFacebook.m#L56-L71), it will be in SDK31

@omrihaim I am currently looking into these functions, here is a [canny post](https://expo.canny.io/feature-requests/p/facebook-login-through-facebook-app) that you can follow. I am not sure exactly why the Facebook `Login Button` exists. Currently you can login to Facebook with `Expo.Facebook`, in Expo this will always force the web `behavior` on iOS. However I am in the process of changing that so you can use native login. I also think that on Android, the native behavior is the default 🙃 
Also using vector icons (included in Expo) you can create the Facebook button very easily: https://github.com/oblador/react-native-vector-icons#iconbutton-component

**Comment by omrihaim** [comment]
Thank you you guys for all your help, and for this project in general.

**Comment by poptocrack** [comment]
@EvanBacon any ETA on the native login for iOS? Thanks!
